e2e/tests/admin_api/get_users_by_standard_attribute_test.go

```diff
@@ -0,0 +1,290 @@
+package adminapi
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/authgear/authgear-server/e2e/pkg/e2eclient"
+	"github.com/authgear/authgear-server/e2e/pkg/testrunner"
+)
+
+func TestGetUsersByStandardAttributeByEmail(t *testing.T) {
+	cmd, err := testrunner.NewEnd2EndCmd(testrunner.NewEnd2EndCmdOptions{
+		TestCase: &testrunner.TestCase{
+			Path: "admin_api/get_users_by_standard_attribute_test.go",
+		},
+		Test: t,
+	})
+	if err != nil {
+		t.Fatalf(err.Error())
+	}
+
+	// Prepare testing user
+	if err := cmd.ExecuteSQLInsertUpdateFile("../../tests/admin_api/get_users_admin_api_user.sql"); err != nil {
+		t.Fatalf(err.Error())
+	}
+
+	resp, err := cmd.Client.GraphQLAPI(nil, nil, cmd.AppID, e2eclient.GraphQLAPIRequest{
+		Query: `
+			query {
+				getUsersByStandardAttribute(attributeName:"email", attributeValue: "e2e_admin_api_get_users@example.com") {
+						id
+						createdAt
+						standardAttributes
+						lastLoginAt
+				}
+			}
+
+		`,
+	})
+	if err != nil {
+		t.Fatalf(err.Error())
+	}
+	if resp.Errors != nil {
+		t.Fatalf("error: %v", resp.Errors)
+	}
+
+	var doc map[string]any
+	if err := json.Unmarshal(resp.Data, &doc); err != nil {
+		panic(err)
+	}
+
+	result := doc["getUsersByStandardAttribute"].([]interface{})
+
+	if len(result) == 0 {
+		t.Fatal("result should not be empty")
+	}
```

Comment:
I prefer we use convey in this test too.

Let me come to you and show you.

Reply:
updated to use Convey